spec/MC_ffg.tla

```diff
@@ -1,5 +1,8 @@
 ----------------------------- MODULE MC_ffg -----------------------------
 
+Nodes == { "A", "B", "C", "D", "E", "F", "G"}
+MAX_SLOT == 5
```

Comment:
Kinda arbitrary, but why not start with 4 nodes (and maybe more slots)?

Reply:
We came down to 5 slots from 1k -> 100 -> 10 -> 5, since all of the recursion-avoiding folds scale quadratically with `MAX_SLOT ` and it was taking a while.